src/core/server/http/http_server.test.ts

```diff
@@ -1089,12 +1089,12 @@ describe('setup contract', () => {
     });
   });
 
-  describe('#registerOnPreAuth', () => {
+  describe('#registerOnPreRouting', () => {
     test('does not throw if called after stop', async () => {
-      const { registerOnPreAuth } = await server.setup(config);
+      const { registerOnPreRouting } = await server.setup(config);
```

Comment:
NIT: not that these tests are really of any significant use, but for consistency maybe add one for the 'new' `registerOnPreAuth`?